apps/expo/src/app/privacy-policy/index.tsx

```diff
@@ -0,0 +1,225 @@
+          <Text
+            className='text-2xl text-slate-200'
+            style={{
+              fontSize: 40,
+              textAlign: 'center',
+              paddingHorizontal: 10,
+              fontWeight: 'bold',
+            }}
+          >
+            Privacy Policy
+          </Text>
+
+          <View style={{ paddingHorizontal: 30, flexDirection: 'row' }}>
+            <Label text='Legal Document' textColor='#CFCFCF' backgroundColor='#48476D' />
+            <Text> </Text>
+            <Label text='Human-Friendly' textColor='#CFCFCF' backgroundColor='#48476D' />
+          </View>
+
+          <Text
+            className='text-2xl text-slate-200'
+            style={{ fontSize: 15, textAlign: 'center', paddingHorizontal: 10 }}
+          >
+            {'\n'}Effective Date: [4/19/2024]{'\n'}
+          </Text>
+          <View className={tailwindClasses.mainTile}>
+            <Text className='text-2xl text-slate-200' style={{ paddingHorizontal: 10 }}>
+              Welcome to Barbell, your ultimate companion for fitness and workout routines. At
+              Barbell, we are committed to protecting your privacy and ensuring the security of your
+              personal information. This Privacy Policy outlines how we collect, use, disclose, and
+              protect your information when you use our mobile application ("App") and services.
+              {'\n\n'}
+            </Text>
+          </View>
+
+          <View className={tailwindClasses.mainTile}>
+            <Text
+              className='text-2xl text-slate-200'
+              style={{ fontSize: 25, paddingHorizontal: 10 }}
+            >
+              1. Information We Collect
+            </Text>
+
+            <Text className='text-2xl text-slate-200' style={{ paddingHorizontal: 10 }}>
+              {'\n'}a. Personal Information: {'\n\t\t\t'} When you create an account with Barbell,
+              we may collect personal information such as your name, email address, age, gender, and
+              profile picture. Additionally, if you choose to link your Spotify account, we may
+              collect information from Spotify's API, including your current listening activity.
+              {'\n\n'}b. Workout Data:{'\n\t\t\t'} Barbell collects information about your workout
+              activities, including exercise routines, duration, intensity, and progress. This data
+              may be shared with friends and the public by default within the App.{'\n\n'}c. Device
+              Information:
+              {'\n\t\t\t'} We may collect information about your device, including its unique device
+              identifier, operating system, and mobile network information.{'\n\n'}d. Location
+              Information:{'\n\t\t\t'} With your consent, we may collect and process information
+              about your precise or approximate location when you use certain features of the App,
+              such as mapping your running routes.{'\n\n'}
```

Comment:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/7507f15c-347b-4d83-9522-e5898a5fe579)

update the styles as I updated here

Reply:
Thank you this was exactly what i needed.